src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -515,17 +518,7 @@ void AtlasEngine::_handleSettingsUpdate()
         _recreateCellCountDependentResources();
     }
 
-    // !!! NOTE !!!
-    // This will indirectly mark the entire viewport as dirty, but AtlasEngine::_recreateBackend()
-    // may do the same, so make sure that that function stays in sync with what this code does.

```

Comment:
With the code above doing a better job handling everything's-invalid-anyways situations, we now don't need to be careful anymore (unless I wrote a bug, but that would be on accident).